### PR TITLE
Updated build instructions for linux for Fedora

### PR DESCRIPTION
**Pull request by RyzenDew**
Added Fedora deps

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -31,4 +31,14 @@ For ubuntu and most derivatives:
...
 8) `ninja`
 
+For Fedora and most derivatives:
+
+1) Make sure vulkansdk is installed and the VULKAN_SDK environment variable is set correctly.
+2) `sudo dnf in -y gtk3-devel libsecret-devel libgcrypt-devel systemd-devel freeglut-devel clang12-libs nasm cmake vcpkg`
+3) Run `git clone --recursive https://github.com/cemu-project/Cemu`
+4) `cd Cemu`
+5) `mkdir build && cd build`
+6) `cmake .. -DCMAKE_BUILD_TYPE=release -DCMAKE_C_COMPILER=/usr/bin/clang-12 -DCMAKE_CXX_COMPILER=/usr/bin/clang++-12 -G Ninja -DCMAKE_MAKE_PROGRAM=/usr/bin/ninja`
```
Comment (leifliddy) [inline BUILD.md:41]: What provides ```/usr/bin/clang-12``` and ```/usr/bin/clang++-12```?

Comment (cobalt2727) [reply BUILD.md:41]: I would imagine `clang12-libs` handles that

Comment (leifliddy) [reply BUILD.md:41]: it doesn't 
```
rpm -ql clang12-libs | grep /bin
```
Interestingly enough the ```clang12``` package is being built
https://koji.fedoraproject.org/koji/packageinfo?packageID=34247
It's just not ending up in the Fedora repo. 
Going to try rebuilding ```clang12-12.0.1-4.fc36.src.rpm```

File: BUILD.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ For Fedora and most derivatives:
 3) Run `git clone --recursive https://github.com/cemu-project/Cemu`
 4) `cd Cemu`
 5) `mkdir build && cd build`
-6) `cmake .. -DCMAKE_BUILD_TYPE=release -DCMAKE_C_COMPILER=/usr/bin/clang-12 -DCMAKE_CXX_COMPILER=/usr/bin/clang++-12 -G Ninja -DCMAKE_MAKE_PROGRAM=/usr/bin/ninja`
+6) `cmake .. -DCMAKE_BUILD_TYPE=release -DCMAKE_C_COMPILER=/usr/bin/clang -DCMAKE_CXX_COMPILER=/usr/bin/clang++ -G Ninja -DCMAKE_BUILD_WITH_INSTALL_RPATH=ON -DCMAKE_MAKE_PROGRAM=/usr/bin/ninja`
 8) `ninja`
```
Comment (leifliddy) [inline BUILD.md:43]: You should probably include ```clang``` as a package to be installed
Do we still need ```clang12-libs```?

File: BUILD.md (hunk 1 above)
Comment (cobalt2727) [reply BUILD.md]: huh, TIL

**Pull request closed without merging**